Release 2.9.2 (#5199)
timescale/timescaledb: CHANGELOG.md

**Bugfixes**
* #4926 Fix corruption when inserting into compressed chunks

## 2.9.2 (2023-01-18)

This release contains bug fixes since the 2.9.1 release.
We recommend that you upgrade at the next available opportunity.

**Bugfixes**

> * #5180 Fix dist_cagg flaky test

Reviewer: Could you check the ordering of the entries? It looks like they are not sorted correctly.

Author: Right, now ordered it correctly.

* #5170 Fix CAgg on CAgg variable bucket size validation
* #5114 Fix issue with deleting data node and dropping database
* #5133 Fix CAgg on CAgg different column order on the original hypertable
* #5152 Fix adding column with NULL constraint to compressed hypertable
* #5181 Fix ChunkAppend, ConstraintAwareAppend child subplan
* #5193 Fix repartition behavior when attaching data node

**Thanks**

> * @ikkala for reporting error when adding column with NULL constraint to compressed hypertable

Reviewer: Please check if we have a `Thanks` for all reported and closed bugs. For example, for #5193 this is missing.

Author: #5193 is internal.
Added a thank note for #5181

**Bugfixes**
* #4926 Fix corruption when inserting into compressed chunks

## 2.9.2 (2023-01-18)

This release contains bug fixes since the 2.9.1 release.
We recommend that you upgrade at the next available opportunity.

**Bugfixes**

> * #5114 Fix issue with deleting data node and dropping database

Reviewer 2: ```suggestion
* #5114 Fix issue with deleting a data node and dropping the database on multi-node
```

> * #5133 Fix CAgg on CAgg using different column order on the original hypertable

Reviewer 2: ```suggestion
* #5133 Fix creating a CAgg on a CAgg where the time column is in a different order of the original hypertable
```

> * #5133 Fix CAgg on CAgg different column order on the original hypertable

Reviewer 3: ```suggestion
* #5133 Fix CAgg on CAgg using different column order than the original hypertable
```

* #5152 Fix adding column with NULL constraint to compressed hypertable
* #5170 Fix CAgg on CAgg variable bucket size validation

> * #5180 Fix default data node availability status

Reviewer 2: ```suggestion
* #5180 Fix default data node availability status on multi-node
```

> * #5181 Fix ChunkAppend, ConstraintAwareAppend child subplan

Reviewer 3: ```suggestion
* #5181 Fix ChunkAppend and ConstraintAwareAppend with TidRangeScan child subplan
```

> * #5193 Fix repartition behavior when attaching data node

Reviewer 2: ```suggestion
* #5193 Fix repartition behavior when attaching data node on multi-node
```
